modules/ROOT/pages/endpoints.adoc

```diff
@@ -4,17 +4,17 @@ Endpoints define the transport protocol configurations necessary to receive or s
 
 Partner Manager uses endpoints to:
 
-* Receive transmissions from partners 
+* Receive transmissions from partners
 * Receive application messages from the backend system
-* Specify the final destination of the message 
+* Specify the final destination of the message
 * Define transport protocols
 * Specify where to send acknowledgments
 
 == Endpoint Ownership
 
```

Comment:
Hi Emilee!

You will get used to this, but just to explain...: 

I am not only an editor-editor but also a reader-advocate editor who always looks for the simplest wording...or the simplest wording to ME, anyway, as someone coming to the text for the very first time. 

Your audience is likely to be more technical than I am, but I will err on the side of caution and make a lot of suggestions, just to make sure.

These are just suggestions. You are absolutely the owner of your topics and The Decider of the final wording, based on your superior understanding of both what you intend to say and who is consuming what you say.

But do please take a moment to consider the alternatives I offer as part of Content and Usability (CU) edits.

Copyedits are a different matter; you should follow them or explain to me why you think I am wrong, if you thing I am wrong.  ;-)

We can discuss this further and customize our working relationship to what works best for you as we go along. I am here first for the audience, but first-and-a-half for YOU, to make you life as easy as possible.